OracleWebLogic/samples/1221-multihost/create-container.sh

```diff
@@ -7,7 +7,8 @@
 . ./setenv.sh
 
 uuid=$(uuidgen)
-name=$prefix-instance-$uuid
+#Define unique server name using suffix entered by user
+name=server$2
```

Comment:
$name is used as the container host name and the server host name.  In the HttpClusterServlet web.xml, calls to the servers using the host name server1 or server2
  <servlet>
    <servlet-name>HttpClusterServlet</servlet-name>
    <servlet-class>weblogic.servlet.proxy.HttpClusterServlet</servlet-class>
    <init-param>
      <param-name>WebLogicCluster</param-name>
      <param-value>server1:7001|server2:7001</param-value>
    </init-param>
    <init-param>